### ISSUE TITLE
case of interface keys

### ISSUE THREAD

**Issue by dmoose** [kristiandupont/kanel#8]
I looked through the source and didn't see an easy way to change the case of the output interfaces.  It appears to always use camel case.  To have the types match the returned values from the database it would be useful to have an option to have the interface keys match the db column names so that a column created_at is not remapped to createdAt.  Otherwise you can't type the database returned values to the generated interfaces.

**Comment by kristiandupont** [comment]
That's correct, I assumed (I guess wrongly) that everyone would want types that match Typescript's standards. Would you want to preserve both class and field names?

**Comment by dmoose** [comment]
I think the class names make sense as standard camel case, but changing the field names makes it hard to use the generated files directly with the db return values.  I think an option to maintain the db field names exactly as they appear in the db would be very helpful.

**Comment by kristiandupont** [comment]
I've published a new version that preserved column names. Let me know if it works for you.

**Comment by dmoose** [comment]
Thanks, I really appreciate your library.  Makes typing the db queries much more automatic than my previous method.
